main.nf

```diff
@@ -1,278 +1,2408 @@
+                                            Available: circexplorer2, circrna_finder, ciriquant, dcc, find_circ, mapsplice.
+                                            Default: ${params.tool}
+
+      --input_type                    [str] Specify the input data type.
+                                            Avaialable: fastq or bam.
+                                            Default: ${params.input_type}
+
+      --input                        [file] Either paths to FASTQ/BAM data (must be surrounded with quotes).
+                                            Indicate multiple files with a suitable wildcard glob pattern i.e '*_{1,2}' for FASTQ paired end reads.
+
+                                            OR
+
+                                            A path to a CSV file (ending .csv) containing file URL/paths and sample IDs.
+                                            Please see online documentation for a template.
+
+      --phenotype                    [file] When differential_expression is provided to --module, a phenotype CSV file (ending .csv) must be provided.
+                                            Sample IDs and response + explanatory variables from metadata must be included. Do not include irrelavant metadata.
+                                            This file is used to construct the DESeq2 model design.
+                                            The response variable must be named 'condition' &  wild-type/control/normal samples named 'control'.
+                                            Please see online documentation for examples of a valid phenotype.csv file.
+
+    circRNA filtering
+      --bsj_reads                     [int] Define the required number of reads spanning circRNA back-splice junction for circRNAs.
+                                            circRNAs with counts below [int] are discarded.
+                                            Disable by setting to 0.
+                                            Default: ${params.bsj_reads}
+
+      --tool_filter                   [int] Specify the minimum number of tools circRNAs must be called by.
+                                            Disable by setting to 0/1 (i.e take the union, do not apply filter).
+                                            Default: ${params.tool_filter}
+
+    Reference files
+      --genome_version                [str] When running the pipeline for the first time, specify the genome version to download for the analysis.
+                                            Gencode reference Fasta, GTF and annotation text files will be automatically generated.
+                                            Available: 'GRCh37', 'GRCh38'.
+
+      --fasta                        [file] Path to Gencode reference genome FASTA file. Must end with '.fa', '.fasta'.
+
+      --gencode_gtf                  [file] Path to Gencode reference GTF file. Must end with '.gtf'.
```

Comment:
wonder if we could use just `--gtf` here, but I guess it only works with gencode?